### ISSUE TITLE
Slow Training Speed On LVIS_V1 DataSet.

### ISSUE THREAD

**Issue by yangJirui** [open-mmlab/mmdetection#7439]
In the beginning, all times (**_time_** and **_data_time_**) looks OK. 
While, at a certain point of Epoch 1, the training speed gradually slow down ( especially the _**data_time**_). 
I'm sure that there were no other operations during training, and i have reproduced this issue many times.

I have trained Mask RCNN on COCO Dataset, but did not reproduce this Issue. It seems that this Issue only exists in Lvis V1 DataSet.

**Detailed LOG**

2022-03-17 11:56:00,123 - mmdet - INFO - Epoch [1][250/7674]    lr: 9.970e-03, eta: 10:43:36, time: 0.441, data_time: 0.089, memory: 5023, loss_rpn_cls: 0.2425, loss_rpn_bbox: 
2022-03-17 11:56:21,294 - mmdet - INFO - Epoch [1][300/7674]    lr: 1.197e-02, eta: 10:44:01, time: 0.424, data_time: 0.059, memory: 5210, loss_rpn_cls: 0.2138, loss_rpn_bbox: 
2022-03-17 11:56:42,652 - mmdet - INFO - Epoch [1][350/7674]    lr: 1.397e-02, eta: 10:45:00, time: 0.427, data_time: 0.048, memory: 5210, loss_rpn_cls: 0.1998, loss_rpn_bbox: 
2022-03-17 11:57:05,258 - mmdet - INFO - Epoch [1][400/7674]    lr: 1.596e-02, eta: 10:50:31, time: 0.453, data_time: 0.047, memory: 6086, loss_rpn_cls: 0.1857, loss_rpn_bbox: 
2022-03-17 11:57:26,114 - mmdet - INFO - Epoch [1][450/7674]    lr: 1.796e-02, eta: 10:48:38, time: 0.417, data_time: 0.060, memory: 6086, loss_rpn_cls: 0.1690, loss_rpn_bbox: 
2022-03-17 11:57:46,272 - mmdet - INFO - Epoch [1][500/7674]    lr: 1.996e-02, eta: 10:45:03, time: 0.404, data_time: 0.061, memory: 6086, loss_rpn_cls: 0.1654, loss_rpn_bbox: 
2022-03-17 11:58:08,190 - mmdet - INFO - Epoch [1][550/7674]    lr: 2.000e-02, eta: 10:46:53, time: 0.438, data_time: 0.069, memory: 6086, loss_rpn_cls: 0.1696, loss_rpn_bbox: 
2022-03-17 11:58:27,782 - mmdet - INFO - Epoch [1][600/7674]    lr: 2.000e-02, eta: 10:42:24, time: 0.392, data_time: 0.046, memory: 6086, loss_rpn_cls: 0.1519, loss_rpn_bbox: 
2022-03-17 11:58:47,586 - mmdet - INFO - Epoch [1][650/7674]    lr: 2.000e-02, eta: 10:39:03, time: 0.396, data_time: 0.057, memory: 6086, loss_rpn_cls: 0.1531, loss_rpn_bbox: 
2022-03-17 11:59:08,557 - mmdet - INFO - Epoch [1][700/7674]    lr: 2.000e-02, eta: 10:38:44, time: 0.420, data_time: 0.068, memory: 6086, loss_rpn_cls: 0.1578, loss_rpn_bbox: 
2022-03-17 11:59:29,379 - mmdet - INFO - Epoch [1][750/7674]    lr: 2.000e-02, eta: 10:38:07, time: 0.417, data_time: 0.043, memory: 6086, loss_rpn_cls: 0.1591, loss_rpn_bbox: 
2022-03-17 11:59:55,387 - mmdet - INFO - Epoch [1][800/7674]    lr: 2.000e-02, eta: 10:47:21, time: 0.520, data_time: 0.056, memory: 8637, loss_rpn_cls: 0.1415, loss_rpn_bbox: 
2022-03-17 12:00:16,554 - mmdet - INFO - Epoch [1][850/7674]    lr: 2.000e-02, eta: 10:46:48, time: 0.423, data_time: 0.051, memory: 8637, loss_rpn_cls: 0.1531, loss_rpn_bbox: 
2022-03-17 12:00:37,603 - mmdet - INFO - Epoch [1][900/7674]    lr: 2.000e-02, eta: 10:46:05, time: 0.421, data_time: 0.057, memory: 8637, loss_rpn_cls: 0.1493, loss_rpn_bbox: 
2022-03-17 12:00:58,241 - mmdet - INFO - Epoch [1][950/7674]    lr: 2.000e-02, eta: 10:44:46, time: 0.413, data_time: 0.070, memory: 8637, loss_rpn_cls: 0.1443, loss_rpn_bbox: 
2022-03-17 12:01:18,652 - mmdet - INFO - Exp name: mask_rcnn_r50_fpn_sample1e-3_mstrain_1x_lvis_v1.py
2022-03-17 12:01:18,652 - mmdet - INFO - Epoch [1][1000/7674]   lr: 2.000e-02, eta: 10:43:09, time: 0.408, data_time: 0.058, memory: 8637, loss_rpn_cls: 0.1381, loss_rpn_bbox: 
2022-03-17 12:01:41,131 - mmdet - INFO - Epoch [1][1050/7674]   lr: 2.000e-02, eta: 10:44:41, time: 0.450, data_time: 0.067, memory: 8637, loss_rpn_cls: 0.1477, loss_rpn_bbox: 
2022-03-17 12:02:01,694 - mmdet - INFO - Epoch [1][1100/7674]   lr: 2.000e-02, eta: 10:43:23, time: 0.411, data_time: 0.046, memory: 8637, loss_rpn_cls: 0.1335, loss_rpn_bbox: 
2022-03-17 12:02:24,039 - mmdet - INFO - Epoch [1][1150/7674]   lr: 2.000e-02, eta: 10:44:31, time: 0.447, data_time: 0.068, memory: 8637, loss_rpn_cls: 0.1466, loss_rpn_bbox: 
2022-03-17 12:02:48,095 - mmdet - INFO - Epoch [1][1200/7674]   lr: 2.000e-02, eta: 10:47:41, time: 0.481, data_time: 0.057, memory: 8637, loss_rpn_cls: 0.1462, loss_rpn_bbox: 
2022-03-17 12:03:24,253 - mmdet - INFO - Epoch [1][1250/7674]   lr: 2.000e-02, eta: 11:05:14, time: 0.723, data_time: 0.243, memory: 8637, loss_rpn_cls: 0.1370, loss_rpn_bbox: 
2022-03-17 12:04:55,400 - mmdet - INFO - Epoch [1][1300/7674]   lr: 2.000e-02, eta: 12:25:23, time: 1.823, data_time: 0.187, memory: 8637, loss_rpn_cls: 0.1379, loss_rpn_bbox: 
2022-03-17 12:07:13,825 - mmdet - INFO - Epoch [1][1350/7674]   lr: 2.000e-02, eta: 14:32:28, time: 2.769, data_time: 1.395, memory: 8637, loss_rpn_cls: 0.1413, loss_rpn_bbox: 
2022-03-17 12:09:28,457 - mmdet - INFO - Epoch [1][1400/7674]   lr: 2.000e-02, eta: 16:26:11, time: 2.692, data_time: 0.123, memory: 8637, loss_rpn_cls: 0.1338, loss_rpn_bbox: 
2022-03-17 12:11:49,853 - mmdet - INFO - Epoch [1][1450/7674]   lr: 2.000e-02, eta: 18:18:57, time: 2.828, data_time: 0.357, memory: 8637, loss_rpn_cls: 0.1293, loss_rpn_bbox: 
2022-03-17 12:15:16,950 - mmdet - INFO - Epoch [1][1500/7674]   lr: 2.000e-02, eta: 21:10:11, time: 4.142, data_time: 0.528, memory: 8637, loss_rpn_cls: 0.1395, loss_rpn_bbox: 
2022-03-17 12:19:06,483 - mmdet - INFO - Epoch [1][1550/7674]   lr: 2.000e-02, eta: 1 day, 0:11:59, time: 4.591, data_time: 0.913, memory: 8637, loss_rpn_cls: 0.1275, loss_rpn_
2022-03-17 12:21:45,274 - mmdet - INFO - Epoch [1][1600/7674]   lr: 2.000e-02, eta: 1 day, 1:55:31, time: 3.176, data_time: 0.538, memory: 8637, loss_rpn_cls: 0.1253, loss_rpn_
2022-03-17 12:25:31,306 - mmdet - INFO - Epoch [1][1650/7674]   lr: 2.000e-02, eta: 1 day, 4:34:02, time: 4.521, data_time: 0.666, memory: 8637, loss_rpn_cls: 0.1294, loss_rpn_
2022-03-17 12:28:27,537 - mmdet - INFO - Epoch [1][1700/7674]   lr: 2.000e-02, eta: 1 day, 6:18:52, time: 3.524, data_time: 0.737, memory: 8637, loss_rpn_cls: 0.1299, loss_rpn_
2022-03-17 12:32:25,119 - mmdet - INFO - Epoch [1][1750/7674]   lr: 2.000e-02, eta: 1 day, 8:50:19, time: 4.751, data_time: 0.457, memory: 8637, loss_rpn_cls: 0.1342, loss_rpn_
2022-03-17 12:34:52,432 - mmdet - INFO - Epoch [1][1800/7674]   lr: 2.000e-02, eta: 1 day, 9:57:42, time: 2.947, data_time: 1.005, memory: 8637, loss_rpn_cls: 0.1268, loss_rpn_
2022-03-17 12:38:59,716 - mmdet - INFO - Epoch [1][1850/7674]   lr: 2.000e-02, eta: 1 day, 12:22:33, time: 4.946, data_time: 1.230, memory: 8637, loss_rpn_cls: 0.1276, loss_rpn
2022-03-17 12:42:29,645 - mmdet - INFO - Epoch [1][1900/7674]   lr: 2.000e-02, eta: 1 day, 14:10:01, time: 4.198, data_time: 0.453, memory: 8637, loss_rpn_cls: 0.1307, loss_rpn
2022-03-17 12:46:41,078 - mmdet - INFO - Epoch [1][1950/7674]   lr: 2.000e-02, eta: 1 day, 16:23:46, time: 5.029, data_time: 0.680, memory: 8637, loss_rpn_cls: 0.1182, loss_rpn
2022-03-17 12:49:55,960 - mmdet - INFO - Exp name: mask_rcnn_r50_fpn_sample1e-3_mstrain_1x_lvis_v1.py

**Reproduction**

1. Command :
```none
sh tools/dist_train.sh configs/lvis/mask_rcnn_r50_fpn_sample1e-3_mstrain_1x_lvis_v1.py 8
```
2. Dataset:
    LVIS_V1

**Environment**
sys.platform: linux
Python: 3.7.6 (default, Jan  8 2020, 19:59:22) [GCC 7.3.0]
CUDA available: True
GPU 0,1,2,3,4,5,6,7: Tesla V100-SXM2-16GB
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 10.2, V10.2.89
GCC: gcc (GCC) 5.2.0
PyTorch: 1.6.0
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2019.0.5 Product Build 20190808 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.5.0 (Git Hash e2ac1fac44c5078ca927cb9b90e1b3066a0b2ed0)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.2
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75
  - CuDNN 7.6.5
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_VULKAN_WRAPPER -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.7.0
OpenCV: 4.3.0
MMCV: 1.4.6
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 10.2
MMDetection: 2.22.0+6b87ac2

**Comment by yangJirui** [comment]
Is there any progress? @RangiLyu
